etl/helpers.py

```diff
@@ -406,6 +409,18 @@ def load_dependency(
 
         return dataset
 
+    def load_snapshot_dependency(self) -> Snapshot:
```

Comment:
Nice! You could change the walkthrough templates to use these functions instead of the generic `load_dependency`.

Reply:
Oh right, gonna refactor it in the future PR.